lib/src/records/list_change_record.dart

```diff
@@ -52,8 +52,8 @@ class ListChangeRecord<E> implements ChangeRecord {
   /// If [addedCount] is not specified it defaults to `removed.length`.
   ListChangeRecord.replace(this.object, this.index, List<E> removed,
       [int addedCount])
-      : this.removed = freezeInDevMode<E>(removed),
```

Comment:
Curious about this one too, does the unnecessary this lint highlight the usage here? Personally I find it a little more clear to have `this` on the LHS but I'm fine with following the lint.

Reply:
Yeah, the linter doesn't like the unnecessary this.